dvc/utils/__init__.py

```diff
@@ -262,7 +262,12 @@ def fix_env(env=None):
             # NOTE: py2 doesn't like unicode strings in environ
             env[cast_bytes_py2(lp_key)] = cast_bytes_py2(lp_orig)
         else:
-            env.pop(lp_key, None)
+            # We used to delete "LD_LIBRARY_PATH" key. We cannot do that
+            # because GitPython uses env to "update" his own internal state.
```

Comment:
Why quotes though? `update` here is used in a litteral sense, no?
```suggestion
            # because GitPython uses env to update his own internal state.
```